src/main/java/seedu/address/model/Model.java

```diff
@@ -100,4 +101,11 @@ public interface Model {
      * @throws NullPointerException if {@code predicate} is null.
      */
     void updateFilteredPersonList(Predicate<Person> predicate);
+
+    /**
+     * Sorts the filtered person list by the given {@code predicate}.
+     * @throws NullPointerException if {@code predicate} is null.
+     */
+    void sortPersonList(Comparator<Person> personComparator);
```

Comment:
JavaDoc for sortPersonList seems to be slightly inaccurate, currently listed as predicate but takes in a comparator.